src/features/advanced-content-insights/AdvancedContentInsightsToggle.scss

```diff
@@ -0,0 +1,12 @@
+.AdvancedContentInsightsToggle {
+    display: flex;
+
+    .toggle-container {
+        margin-bottom: 0;
+    }
+}
+
+.AdvancedContentInsightsToggle-icon {
+    display: inline;
```

Comment:
is it okay if we change to `inline-block`? I think margin's and padding's are less predictable (maybe don't work?) on inline elements